errorScaleCal/test/readNtuples.C

```diff
@@ -12,71 +12,83 @@
 
 #include "BeamSpotCalibration/errorScaleCal/src/pvTree.h"
 
-// set max n_tracks per sub vtx (vtx/2)
-int max_n_tracks   = 80;//80
-int max_n_vertices = 40;//40
-int max_sum_pt     = 20;
+int max_n_tracks   = 35;
+int max_n_vertices = 6;
+int max_sum_pt     = 25;
 int max_res        = 10;
+int sum_pt_step    = 2;
```

Comment:
The previous default was actually unclear because it was defined inside the for-loop that populates the histograms ending up in many created but empty histograms. Now only histograms corresponding to pt step are created and filled (0 corresponds to 0-2 GeV sum-pt, 2 corresponds to 2-4 GeV and so on).2 GeV step seemed an appropriate value. Can be increased for 13 TeV collisions of course.

Reply:
From the previous `for (int i = 0; i < max_sum_pt; i ++)` I understood it was by default using steps of 1 (GeV). But ok using steps of 2 GeV should be absolutely fine and it can always be customized in the future if needed.

Reply:
Histograms where created with a 1 GeV step you are right but in the filling step only some of them where filled and this is not correct. See line 125-126.

Reply:
Ok, reading back lines 125-126 as you suggested, I noticed another (possible) error:
```
      for (int i=0; i < max_sum_pt; i += sum_pt_step){ 
        if (thePV.sumPt_subVtx1 > i*sum_pt_step && thePV.sumPt_subVtx1 < i*sum_pt_step+sum_pt_step &&
            thePV.sumPt_subVtx2 > i*sum_pt_step && thePV.sumPt_subVtx2 < i*sum_pt_step+sum_pt_step 
           ){
```
Won't this generate intervals: [0-2], [4-6], [8-10]... (given that `sum_pt_step = 2`) ?
The first line maybe should be replacedy by:
```
for (int i = 0; i < max_sum_pt; i ++)
```

Reply:
Yes you're right, but we preferred to fix the if statement.

Reply:
the if statement is indeed correct (bins of 2 GeV instead of bins of 10 GeV), but the for sttement should be fixed as well I think

Reply:
fixed in 1454d7d